### PR TITLE
Switch sample to use distroless base.

### PR DESCRIPTION
**Pull request by mattmoor**

### CODE REVIEW

**Comment by mattmoor** [comment]
@dlorenc PTAL

File: README.md [hunk 1]
```diff
@@ -50,8 +50,7 @@ docker_repositories()
 docker_pull(
   name = "java_base",
   registry = "gcr.io",
-  # TODO(mattmoor): Replace this.
-  repository = "google-appengine/java",
+  repository = "distroless/java",
   # 'tag' is also supported, but digest is encouraged for reproducibility.
   digest = "sha256:deadbeef",
```
Comment (dlorenc) [inline README.md]: Want to use a real digest here?

Comment (mattmoor) [inline README.md]: No, because people copy and paste :)

Comment (pcj) [reply README.md]: Can we leave a note as to the recommended way to obtain the sha256 of the pulled image?

Comment (mattmoor) [reply README.md]: You should be able to put `gcr.io/foo/bar:blah` into your browser, and then clicking "Show Pull Command" will contain the digest.

Comment (pcj) [reply README.md]: Thanks.  Attaching screenshot for someone in the future.  Might be nice to include this info in the README.

screen shot 2017-08-26 at 3 58 25 pm

**Pull request merged**